playground/nextjs-app-router/components/AppProvider.tsx

```diff
@@ -228,6 +228,11 @@ export const AppProvider = ({ children }: { children: React.ReactNode }) => {
             theme: componentTheme === 'none' ? undefined : componentTheme,
           },
           paymaster: paymasters?.[chainId || 8453]?.url,
+          wallet: {
+            display: 'modal',
+            termsUrl: 'https://www.coinbase.com/legal/cookie', // URL to the terms of service for the wallet modal
+            privacyUrl: 'https://www.coinbase.com/legal/privacy', // URL to the privacy policy for the wallet modal
+          },
```

Comment:
do we also want to give playground users the ability to see `wallet.display.classic`?